sdcm/utils/issues.py

```diff
@@ -0,0 +1,93 @@
+import re
+import sys
+import logging
+from functools import lru_cache
+from dataclasses import dataclass
+
+import github
+import github.Auth
+from github.GithubException import UnknownObjectException
+
+from sdcm.keystore import KeyStore
+from sdcm.sct_config import SCTConfiguration
+from sdcm.sct_events.base import Severity
+from sdcm.sct_events.system import TestFrameworkEvent
+
+scylla_issue_pattern = re.compile(
+    r"^\s*((((?P<user_id>[\w:-]+)/)?(?P<repo_id>[\w:-]+))?#)?(?P<issue_id>\d+)\s*$", re.IGNORECASE)
+
+scylla_issue_https_pattern = re.compile(
+    r"^https?:/.*?/(?P<user_id>[\w:-]+)/(?P<repo_id>[\w:-]+)/.*?/(?P<issue_id>\d+?)$", re.IGNORECASE)
+
+
+@dataclass
+class Issue:
+    user_id: str | None
+    repo_id: str | None
+    issue_id: int | None
+
+
+class SkipPerIssues:
```

Comment:
This class doesn't handle the `skip`.
So, I think, it is incorrect to use that word for it.
What about `GithubIssue`?

Reply:
it's does hold the logic about skipping, and that is it's sole purpose.
it's not a general purpose issue with it's details.